drivers/drivers.go

```diff
@@ -0,0 +1,285 @@
+// Package drivers abstracts different object storages, such as local, s3
```

Comment:
OMG just realized that the original library already had this "object storage" term as well. Now I'm confused lol
But anyway, might wanna keep it as store for now.
```suggestion
// Package drivers abstracts different object stores, such as local, s3
```